src/test/java/edu/kis/powp/jobs2d/TestJobs2dApp.java

```diff
@@ -55,16 +56,17 @@ private static void setupCommandTests(Application application) {
 	 * @param application Application context.
 	 */
 	private static void setupDrivers(Application application) {
-		Job2dDriver loggerDriver = new LoggerDriver();
-		DriverFeature.addDriver("Logger driver", loggerDriver);
+		DriverComposite driverComposite = new DriverComposite();
+		driverComposite.add(new LoggerDriver());
+		DriverFeature.addDriver("Logger driver", driverComposite.getDriver(0));
 
 		DrawPanelController drawerController = DrawerFeature.getDrawerController();
-		Job2dDriver driver = new LineDriverAdapter(drawerController, LineFactory.getBasicLine(), "basic");
-		DriverFeature.addDriver("Line Simulator", driver);
-		DriverFeature.getDriverManager().setCurrentDriver(driver);
+		driverComposite.add(new LineDriverAdapter(drawerController, LineFactory.getBasicLine(), "basic"));
+		DriverFeature.addDriver("Line Simulator", driverComposite.getDriver(1));
```

Comment:
This doesn't make sense to me.

Reply:
Just using composite's list to use drivers.